.github/workflows/lint.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Run Linters
+on:
+  - push
+jobs:
+  linters:
+    name: Run Linters
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
I think we are using 3.6 on azure tests. I'd either use 3.6 here or change it there to 3.8 as well.

Reply:
done.